src/Monitor/Monitor/help/New-AzAutoscaleRule.md

```diff
@@ -274,7 +274,7 @@ Aliases:
 
 Required: False
 Position: Named
-Default value: None
+Default value: 1 minute
```

Comment:
What is the impact of this change?  What is the impact now if no value is set.  Whant to make sure this is not a breaking change in behavior.

Reply:
This did not change the behavior of "without value" (or value == 0) Without value should have used the default (5 minutes before, now 1 minute)